src/main/java/duke/Event.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Event extends Task {
+    protected String at;
+    protected LocalDate dateTime;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+        dateTime = LocalDate.parse(at, DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm"));
+        // System.out.println(dateTime.toString());
+
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + "(at: " + at + ")";
+    }
+
+    @Override
+    public String toUser() {
```

Comment:
Perhaps a more intuitive name for this method? And for other overwritten/overwriting methods as well.